### DISCUSSION TITLE
Failing to create site

### DISCUSSION THREAD

**Discussion by Davichet-e** [thegeeklab/hugo-geekdoc#868] [Q&A]
Hello,

Trying to follow the tutorial, I run:
```
hugo new site demosite
cd demosite

mkdir -p themes/hugo-geekdoc/
curl -L https://github.com/thegeeklab/hugo-geekdoc/releases/latest/download/hugo-geekdoc.tar.gz | tar -xz -C themes/hugo-geekdoc/ --strip-components=1

<write to config.toml the corresponding config>

hugo server -D
```

And I get this error:
```
Error: add site dependencies: load resources: loading translations: "/home/user/test/themes/hugo-geekdoc/i18n/am.yaml:2:1": failed to load translations: failed to unmarshal am.yaml: yaml: unmarshal errors:
  line 2: cannot unmarshal !!map into []map[string]interface {}
```

How can I solve this? Am I missing something? Thank you in advance.

OS: Ubuntu 20.04.6 LTS (WSL)
Hugo version: v0.68.3/extended linux/amd64 BuildDate: 2020-03-25T06:15:45Z (I just installed it from apt)

**Comment by Davichet-e** [comment]
I just realised that the last Hugo version is 0.133. I just installed Hugo from apt, so probably it's an issue on the registry of WSL for Ubuntu having a very old version, I'll try to install the latest Hugo version and confirm that's the issue

**Comment by xoxys** [comment]
Yes, most likely that's related to the outdated Hugo version.
